project1/styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+    background-color: rgb(255, 220, 249);
+    font-family: 'Cute Font', cursive;
+    font-family: 'Lato', sans-serif;
+    font-family: 'Roboto', sans-serif;
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+main {
+    /* border: 10px solid #444; */
+
+    width: 1310px;
+    padding: 70px;
+    border: 20px solid rgb(255, 199, 232);
+    background-color: rgb(253, 171, 210);
+    text-align: left;
+    margin: 0 auto;
+}
+
+h1 {
+    color: rgb(255, 255, 255);
+    background-color: rgb(255, 73, 158);
+    text-align: center;
+    font-size: 70px;
+    font-weight: bold;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    font-family: 'Prata', serif;
+    margin-top: 8px;
+    margin-bottom: 8px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+
+}
+
+h2 {
+    font-family: 'Prata', serif;
+    opacity: 0.7;
+    color: deeppink;
+    text-align: right
+}
+
+h3 {
+    font-family: 'Prata', serif;
+    opacity: 0.7;
+    text-align: right;
+    color: deeppink;
+
+}
+
+p {
+    color: deeppink;
+    text-align: ;
+    font-size: 25px;
+    line-height: 35px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    padding-left: 17px;
+    padding-right: 10px padding-bottom: 15px;
```

Comment:
This line is broken because it is missing a semicolon; you want:
```css
    padding-right: 10px;
```
And you can delete the "padding-bottom: 15px;" because you've already added that on line 61.